src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -105,6 +108,78 @@ def set_lint_args(parser: argparse.ArgumentParser) -> argparse.Namespace:
     return parser.parse_args()
 
 
+def get_full_paths(file_list: list) -> list:
+    """
+    Update file paths to be absolute paths with system separators.
+
+    Parameters
+    ----------
+    file_list: list
+        List containing committed files.
+
+    Returns
+    -------
+    list
+        List containing the full paths of committed files.
+    """
+    full_path_files = []
+    for file in file_list:
+        if "win" in sys.platform:
+            split_str = file.split("/")
+            full_path_files.append(os.path.abspath(os.path.join(*split_str)))
+        else:
+            full_path_files.append(os.path.abspath(file))
+
+    return full_path_files
+
```

Comment:
Why not use pathlib? It will handle the paths properly based on your OS

Reply:
Also, with a pathlib.Path object you can request at any moment the absolute path. So no need to have the dedicated function to get "full" paths. Just call the absolute path method in the Path object